src/ring_theory/jacobson.lean

```diff
@@ -587,9 +587,9 @@ begin
     exact (quotient.mk P1).is_integral_of_surjective (quotient.mk_surjective) }
 end
 
-lemma comp_C_integral_of_surjective_of_jacobson {R : Type*} [integral_domain R] [is_jacobson R]
-  {S : Type*} [field S] {n : ℕ} (f : (mv_polynomial (fin n) R) →+* S) (hf : function.surjective f) :
-  (f.comp C).is_integral :=
+private lemma comp_C_integral_of_surjective_of_jacobson' {R : Type*} [integral_domain R]
```

Comment:
I do not find `private` necessary.

Reply:
I think that `comp_C_integral_of_surjective_of_jacobson` is a strictly more general statement than this one, this lemma is just being used to bootsrap that one, so I made it private. It's actually not that hard to inline this though so I've just moved this into the body of `comp_C_integral_of_surjective_of_jacobson`.